### PR TITLE
[MX-141] [MX-144] Adds feedback/privacy request user interface

### PR DESCRIPTION
**Pull request by ashfurrow**
Also updates to use the new Palette Settings icon (see: https://github.com/artsy/palette/pull/625).

I feel like I have some 'splaining to do on this PR 😅 The iOS email compose API is specific enough _to_ iOS that React Native doesn't support it specifically. The only way to open the email compose sheet is to use React Native's `Linking` module with a `mailto:address@example.com?subject=...` URL. Feels weird, but that's fine. The problem is that not all iOS devices are configured with an email account. iOS has a `[MFMailComposeViewController canSendMail]` method that you can query, but no such method existing in React Native. Instead, you have to `.catch()` on the promise returned from `Linking.openURL()`, which felt pretty dirty to me. So I wrote a small native function to use the native `MFMailComposeViewController`.

Here's the new Privacy Request view:

Screen Shot 2020-01-15 at 15 53 20

Tapping "Privacy Policy" will open this modal:

Screen Shot 2020-01-15 at 15 53 28

Tapping on the other links in the simulator will show this error, because the simulator can't open email compose views. It should open the mail compose view when tapped on devices.

Screen Shot 2020-01-15 at 16 07 21

### CODE REVIEW

**Comment by ArtsyOpenSource** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>
<b>🃏 FAIL</b> in <a href="https://github.com/artsy/emission/blob/ccpa_ui/src/lib/Scenes/Settings/__tests__/MyProfile-tests.tsx">MyProfile-tests.tsx</a>


<li>
<a href='https://github.com/artsy/emission/blob/ccpa_ui/src/lib/Scenes/Settings/__tests__/MyProfile-tests.tsx#L10'>looks like expected</a>
<br/>
Snapshot name: `looks like expected 1`- Snapshot+ Received@@ -88,11 +88,11 @@ > <View accessible={true} alignItems="center" flexDirection="row"- focusable={false}+ focusable={true} justifyContent="space-between" onClick={[Function]} onResponderGrant={[Function]} onResponderMove={[Function]} onResponderRelease={[Function]}

<details>
<summary>Full message</summary>
<pre><code>
Error: expect(received).toMatchSnapshot()

Snapshot name: `looks like expected 1`

- Snapshot
+ Received

@@ -88,11 +88,11 @@
      >
        <View
          accessible={true}
          alignItems="center"
          flexDirection="row"
-         focusable={false}
+         focusable={true}
          justifyContent="space-between"
          onClick={[Function]}
          onResponderGrant={[Function]}
          onResponderMove={[Function]}
          onResponderRelease={[Function]}
    at Object.toMatchSnapshot (/home/travis/build/artsy/emission/src/lib/Scenes/Settings/__tests__/MyProfile-tests.tsx:10:16)
    at Object.asyncJestTest (/home/travis/build/artsy/emission/node_modules/jest-jasmine2/build/jasmineAsyncInstall.js:102:37)
    at /home/travis/build/artsy/emission/node_modules/jest-jasmine2/build/queueRunner.js:43:12
    at new Promise (<anonymous>)
    at mapper (/home/travis/build/artsy/emission/node_modules/jest-jasmine2/build/queueRunner.js:26:19)
    at /home/travis/build/artsy/emission/node_modules/jest-jasmine2/build/queueRunner.js:73:41
</code></pre>
</details>
</li>
<br/>

</td>
    </tr>
  </tbody>
</table>




<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

**Comment by brainbicycle** [comment]
> Instead, you have to `.catch()` on the promise returned from `Linking.openURL()`, which felt pretty dirty to me. So I wrote a small native function to use the native `MFMailComposeViewController`.

Agreed this is much better.

**Comment by artsyit** [comment]
:rocket: PR was released in v1.21.8 :rocket:

File: package.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "emission",
   "version": "1.21.7",
-  "native-code-version": 42,
+  "native-code-version": 43,
```
Comment (alloy) [inline package.json]: Tssk tssk 😛

**Pull request merged**